pkgs/development/tools/tokio-console/default.nix

```diff
@@ -6,19 +6,21 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "tokio-console";
-  version = "0.1.7";
+  version = "0.1.9";
 
   src = fetchFromGitHub {
     owner = "tokio-rs";
     repo = "console";
     rev = "tokio-console-v${version}";
-    sha256 = "sha256-yTNLKpBkzzN0X73CjN/UXRGjAGOnCCgJa6A6loA6baM=";
+    sha256 = "sha256-zISgEhUmAfHErq4AelbnSwtKjtxYH//pbLUAlPKxQYk=";
   };
 
-  cargoSha256 = "sha256-K/auhqlL/K6RYE0lHyvSUqK1cOwJBBZD3QTUevZzLXQ=";
+  cargoSha256 = "sha256-qK8U6BZN7sdBP8CbzsDeewsGulNA/KFVS9vscBxysRg=";
```

Comment:
Please change this to cargoHash.

Reply:
Done.

Same question as above, is cargoSha256 soft-deprecated?

Reply:
Yea. `<X>hash` is future-proof - in the case of a deprecation of hashing algorithms (like happened with MD5).